Landing/elements/textCutter/index.tsx

```diff
@@ -0,0 +1,68 @@
+// Core
+import React, {FC, useEffect, useState, useContext} from 'react';
+
+// View
+import * as S from './styles'
+
+// Locale
+import { LocalContext } from '../../app/page'
+
+
+
+interface TextCutterT {
+    children: string
+}
+
+export const TextCutter:FC<TextCutterT>= ({children}) => {
+    const [isOpen, setIsOpen] = useState(false)
+
+    useEffect(()=>{
+        if (children?.length <=400) {
+            setIsOpen(true)
+        }
+    }, [children])
+
+    const { local } = useContext(LocalContext)
+    
+    const ButtonLocale = () => {
+        let btnName 
+
+        switch(local){
```

Comment:
better to get this translation for strapi. If we don't.  Better to create a ENUM on 

```
enum ButtonLocale {
    EN = 'read more',
    PL = 'czytaj więcej',
    UK = 'читати далi',
}

type Locale = keyof typeof ButtonLocale;

// Assuming you have a LocalContext that provides a locale
const { local } = useContext(LocalContext);

const getButtonName = () => {
    const locale = local.toUpperCase() as Locale;
    return ButtonLocale[locale] || ButtonLocale.EN;
}
```

Something like that ( need to check)